pages/style-guide/prose.md

```diff
@@ -0,0 +1,111 @@
+<script>{
+    "title": "Prose Style Guide"
+}</script>
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/).
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
+---
+title:  <article title>
+level:  [beginner|intermediate|advance]
+source:  <url of source of the material derived>
+attribution:
+  - Ralph Whitbeck <ralph@email.com>
+  - John Paul <john@email.com>
+---
+```
```

Comment:
To be honest, If you don't know how the architecture of our sites is setup, you're probably not the right guy to handle this.

----

The thing is that, in the past, we used YAML for front matters in some sites and now we use JSON, but the example in here still uses YAML.

Also, we mean some of the properties you use in your example are specific for the learn site  (`learn.jquery.com`). Those properties are not needed in other sites.

Remember that the original article was specific for `learn.jquery.com` while this must be true for every site.